client/src/main/java/org/eclipse/hono/client/impl/CommandClientImpl.java

```diff
@@ -240,10 +245,11 @@ protected BufferResult getResult(
     /**
      * Creates a new command client for a tenant and device.
      * <p>
-     * The instance created is scoped to the given device.
-     * In particular, the sender link's target address is set to
-     * <em>control/${tenantId}/${deviceId}</em> and the receiver link's source
-     * address is set to <em>control/${tenantId}/${deviceId}/${replyId}</em>.
+     * The instance created is scoped to the given device. In particular, the target address of messages is set to
```

Comment:
didn't we want to change this so that the client is scoped to the tenant only?

Reply:
This topic is discussed in #1255. I don't want to include a solution for that in this PR here.